components/bitly/common/common.mjs

```diff
@@ -0,0 +1,54 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  methods: {
+    async createBitlink($, payload, oauth_access_token) {
```

Comment:
Also, rename `payload` to `data` so you can just pass `data` in the HTTP request below